lib/taskScheduler.js

```diff
@@ -59,6 +60,17 @@ var TaskScheduler = function(config) {
       capabilitiesSpecs.forEach(function(spec) {
         specLists.push([spec]);
       });
+    } else if (capabilities.shardTestSuites){
```

Comment:
Please document this in [docs/referenceConf.js](https://github.com/angular/protractor/blob/master/docs/referenceConf.js).